packaging/installer/kickstart.sh

```diff
@@ -1718,6 +1778,10 @@ while [ -n "${1}" ]; do
       INSTALL_PREFIX="${2}"
       shift 1
       ;;
+    "--install-version")
+      INSTALL_VERSION="${2}"
+      shift 1
+      ;;
```

Comment:
This should also disable auto-updates by default (which unfortunately means that it will have to depend on https://github.com/netdata/netdata/pull/12634 for sane handling of that).